lochness/redcap/__init__.py

```diff
@@ -238,38 +238,44 @@ def get_run_sheets_for_datatypes(api_url, api_key,
 
             content_dict_list = json.loads(content)
 
-            content_num = 0
-            for content_dict in content_dict_list:
+            # select the right form, and ignore the weird empty dictionary
+            content_dict_list = [x for x in content_dict_list if
+                    x[run_sheet_name+'_complete'] != '']
+
+            # for run sheet at each timepoint - baseline, follow up1, etc.
+            # content_num is set to start with 1 to match the session number
+            for content_num, content_dict in enumerate(content_dict_list, 1):
```

Comment:
1. What is a session number here?

2. We discussed in EEG meeting that `content_num` should start with `0` to make sense with `baseline`.

Reply:
1. session number: 1 for first data point (baseline), 2 for second time point, 3 for third timepoint
2. Spero was suggesting we make `0`, and `2` to represent the time of the EEG scan in months from baseline, which we concluded that this would make things more confusing. So starting from `1` should be fine.